tools/adventure-pack/goodies/typescript/Iterator.prototype.toArray/index.test.ts

```diff
@@ -0,0 +1,24 @@
+import "../Iterator.prototype.toArray";
+import { describe, expect, it } from "@jest/globals";
```

Comment:
Eventually I think we will need to add this as a lint rule, but I think a good style for imports is first external imports and then internal imports, separated by an empty line.

Reply:
Ack!